### ISSUE TITLE
Missing import for SadeemQuestionRetrieval 

### ISSUE THREAD

**Issue by PrithivirajDamodaran** [embeddings-benchmark/mteb#1143]
Under retrieval tasks for Arabic language SadeemQuestionRetrieval is not recognised by MTEB. Neither lists nor runs. 

Please advise.

**Comment by KennethEnevoldsen** [comment]
Added a fix in  #1146. It should now run using: 
```python
from __future__ import annotations

import mteb

tasks = mteb.get_tasks(tasks=["SadeemQuestionRetrieval"])
tasks[0].load_data()

model = mteb.get_model("sentence-transformers/paraphrase-MiniLM-L6-v2")

eval= mteb.MTEB(tasks=tasks)
res = eval.run(model)
```

**Comment by PrithivirajDamodaran** [comment]
Thanks, Simple pip upgrade would work or should I install from source?
